### PR TITLE
[ i18n ] Added translation key vis_records_start_end_of_total

### PR DESCRIPTION
**Pull request by litvinovg**
[Companion VIVO PR](https://github.com/vivo-project/VIVO/pull/3791)
# What's new?
New translation key vis_records_start_end_of_total for list of publications at /vis/publication-graph page

# Interested parties
@hauschke 
@chenejac

### CODE REVIEW

File: sr_Latn_RS/webapp/src/main/webapp/i18n/vivo_all_sr_Latn_RS.properties [hunk 1]
```diff
@@ -618,6 +618,7 @@ vis_caching_process = Šta je sve uključeno u proces keširanja?
 vis_tools_note_two = Implementirano je keširanje koje čuva informacije o hierarhiji organizacija -- pre svega koje publikacije pripadaju kojoj organizaciji -- tako što se čuva RDF model u memoriji.
 vis_tools_note_three = Trenutno ove modele keširamo u memoriji. Keš se kreira (samo jednom) prilikom prvog zahteva korisnika nakon restarta servera. Zbog ovog će se koristiti isti model sve dok se server ne restartuje, što znači da podaci u modelu možda neće biti konzistentni u zavisnosti od toga kada su kreirani. Ovo funkcioniše dovoljno dobro za sada. U narednim verzijama aplikacije ćemo unaprediti ovo rešenje tako da se modeli čuvaju na disku i periodično periodično osvežavaju.
 vis_tools_note_four = Modeli se ponovo učitavaju svaki put kad se server restartuje. Sa obzirom da ovo nije praktično u produkciji, administratori mogu da koriste "osvežite keš" link iznad da učitaju ponovo modele bez potrebe za restartom servera.
+vis_records_start_end_of_total = Records _START_ - _END_ of _TOTAL_
```
Comment (chenejac) [inline sr_Latn_RS/webapp/src/main/webapp/i18n/vivo_all_sr_Latn_RS.properties]:
```suggestion
vis_records_start_end_of_total = Zapisi _START_ - _END_ od ukupno _TOTAL_ 
```

**Review by chenejac — CHANGES_REQUESTED** [review]

File: de_DE/webapp/src/main/webapp/i18n/vivo_all_de_DE.properties [hunk 2]
```diff
@@ -615,6 +615,7 @@ vis_caching_process = Was beinhaltet der Vorgang zur Zwischenspeicherung?
...
 vis_tools_note_three = Zurzeit speichern wir diese Modelle im Hauptspeicherbereich zwischen. Der Zwischenspeicher wird (nur einmal) bei der ersten Benutzeranfrage nach einem Neustart des Servers aufgebaut. Daher wird bis zum nächsten Neustart das gleiche Modell bedient. Das bedeutet, dass die Daten in den Modellen veralten können, je nachdem, wann das Modell zum letzten Mal erstellt wurde. Das funktioniert fürs Erste gut. In kommenden Releases werden wir diese Lösung dahingehend verbessern, dass Modelle auf Festplatte gespeichert und regelmäßig aktualisiert werden.
 vis_tools_note_four = Die Modelle werden bei jedem Neustart des Servers aktualisiert. Da dies auf der Produktionsinstanz  allgemein unpraktisch ist, können AdministratorInnen stattdessen den “Zwischenspeicher aktualisieren”-Link oben verwenden, um ohne einen Neustart eine Aktualisierung durchzuführen.
+vis_records_start_end_of_total = Records _START_ - _END_ of _TOTAL_
```
Comment (bkampe) [inline de_DE/webapp/src/main/webapp/i18n/vivo_all_de_DE.properties]:
```suggestion
vis_records_start_end_of_total = Datensatz _START_ - _END_ von _TOTAL_ 
```

**Review by bkampe — CHANGES_REQUESTED** [review]

**Comment by hauschke** [comment]
@bkampe Maybe "Eintrag" instead of "Datensatz"? 

It's been used in the temporal graph visualization:

grafik

File: en_CA/webapp/src/main/webapp/i18n/vivo_all_en_CA.properties [hunk 3]
```diff
@@ -615,6 +615,7 @@ vis_caching_process = What's involved in the caching process?
 vis_tools_note_two = To this end we have devised a caching solution which will retain information about the hierarchy of organizations -- namely, which publications are attributed to which organizations -- by storing the RDF model.
```
Comment (michel-heon) [inline en_CA/webapp/src/main/webapp/i18n/vivo_all_en_CA.properties:615]:
```suggestion
vis_tools_note_two = À cette fin, nous avons conçu une solution de mise en cache qui conservera les informations sur la hiérarchie des organisations ( à savoir, quelles publications sont attribuées à quelles organisations ),  en stockant le modèle RDF.

```

**Comment by AnnaGuillaumet** [comment]
Registros 

(normally the word Registry is not usually used, only the numbers appear, but the translation would be, as above: Registros)

File: en_CA/webapp/src/main/webapp/i18n/vivo_all_en_CA.properties [hunk 4]
```diff
@@ -615,6 +615,7 @@ vis_caching_process = What's involved in the caching process?
 vis_tools_note_two = To this end we have devised a caching solution which will retain information about the hierarchy of organizations -- namely, which publications are attributed to which organizations -- by storing the RDF model.
 vis_tools_note_three = We're currently caching these models in memory.  The cache is built (only once) on the first user request after a server restart.  Because of this, the same model will be served until the next restart. This means that the data in these models may become stale depending upon when it was last created. This works well enough for now. In future releases we will improve this solution so that models are stored on disk and periodically updated.
```
Comment (michel-heon) [inline en_CA/webapp/src/main/webapp/i18n/vivo_all_en_CA.properties:616]:
```suggestion
vis_tools_note_three = Nous mettons actuellement ces modèles dans la mémoire tampon.  La mémoire tampon est construit (une seule fois) lors de la première demande de l'utilisateur après un redémarrage du serveur.  De ce fait, le même modèle sera utilisé jusqu'au prochain redémarrage. Cela signifie que les données de ces modèles peuvent devenir périmées en fonction de la date de leur dernière création. Cela fonctionne assez bien pour l'instant. Dans les prochaines versions, nous améliorerons cette solution afin que les modèles soient stockés sur le disque et mis à jour périodiquement.
```

File: en_CA/webapp/src/main/webapp/i18n/vivo_all_en_CA.properties [hunk 5]
```diff
@@ -615,6 +615,7 @@ vis_caching_process = What's involved in the caching process?
 vis_tools_note_two = To this end we have devised a caching solution which will retain information about the hierarchy of organizations -- namely, which publications are attributed to which organizations -- by storing the RDF model.
 vis_tools_note_three = We're currently caching these models in memory.  The cache is built (only once) on the first user request after a server restart.  Because of this, the same model will be served until the next restart. This means that the data in these models may become stale depending upon when it was last created. This works well enough for now. In future releases we will improve this solution so that models are stored on disk and periodically updated.
 vis_tools_note_four = The models are refreshed each time the server restarts.  Since this is not generally practical on production instances, administrators can instead use the "refresh cache" link above to do this without a restart.
```
Comment (michel-heon) [inline en_CA/webapp/src/main/webapp/i18n/vivo_all_en_CA.properties:617]:
```suggestion
vis_tools_note_four = Les modèles sont rafraîchis chaque fois que le serveur redémarre.  Comme ce n'est généralement pas utile sur les instances de production, les administrateurs peuvent utiliser le lien "refresh cache" ci-dessus pour effectuer cette opération sans redémarrage.


```

File: en_CA/webapp/src/main/webapp/i18n/vivo_all_en_CA.properties [hunk 6]
```diff
@@ -615,6 +615,7 @@ vis_caching_process = What's involved in the caching process?
 vis_tools_note_two = To this end we have devised a caching solution which will retain information about the hierarchy of organizations -- namely, which publications are attributed to which organizations -- by storing the RDF model.
 vis_tools_note_three = We're currently caching these models in memory.  The cache is built (only once) on the first user request after a server restart.  Because of this, the same model will be served until the next restart. This means that the data in these models may become stale depending upon when it was last created. This works well enough for now. In future releases we will improve this solution so that models are stored on disk and periodically updated.
 vis_tools_note_four = The models are refreshed each time the server restarts.  Since this is not generally practical on production instances, administrators can instead use the "refresh cache" link above to do this without a restart.
+vis_records_start_end_of_total = Records _START_ - _END_ of _TOTAL_ 
 capability_map=Capability Map
```
Comment (michel-heon) [inline en_CA/webapp/src/main/webapp/i18n/vivo_all_en_CA.properties:619]:
```suggestion
capability_map=Cartographie des expertises

```

File: en_CA/webapp/src/main/webapp/i18n/vivo_all_en_CA.properties [hunk 7]
```diff
@@ -615,6 +615,7 @@ vis_caching_process = What's involved in the caching process?
 vis_tools_note_two = To this end we have devised a caching solution which will retain information about the hierarchy of organizations -- namely, which publications are attributed to which organizations -- by storing the RDF model.
 vis_tools_note_three = We're currently caching these models in memory.  The cache is built (only once) on the first user request after a server restart.  Because of this, the same model will be served until the next restart. This means that the data in these models may become stale depending upon when it was last created. This works well enough for now. In future releases we will improve this solution so that models are stored on disk and periodically updated.
 vis_tools_note_four = The models are refreshed each time the server restarts.  Since this is not generally practical on production instances, administrators can instead use the "refresh cache" link above to do this without a restart.
+vis_records_start_end_of_total = Records _START_ - _END_ of _TOTAL_ 
 capability_map=Capability Map
 term_capitalized=Term
```
Comment (michel-heon) [inline en_CA/webapp/src/main/webapp/i18n/vivo_all_en_CA.properties:620]:
```suggestion
term_capitalized=Terme

```

File: en_CA/webapp/src/main/webapp/i18n/vivo_all_en_CA.properties [hunk 8]
```diff
@@ -615,6 +615,7 @@ vis_caching_process = What's involved in the caching process?
 vis_tools_note_two = To this end we have devised a caching solution which will retain information about the hierarchy of organizations -- namely, which publications are attributed to which organizations -- by storing the RDF model.
 vis_tools_note_three = We're currently caching these models in memory.  The cache is built (only once) on the first user request after a server restart.  Because of this, the same model will be served until the next restart. This means that the data in these models may become stale depending upon when it was last created. This works well enough for now. In future releases we will improve this solution so that models are stored on disk and periodically updated.
 vis_tools_note_four = The models are refreshed each time the server restarts.  Since this is not generally practical on production instances, administrators can instead use the "refresh cache" link above to do this without a restart.
+vis_records_start_end_of_total = Records _START_ - _END_ of _TOTAL_
```
Comment (michel-heon) [inline en_CA/webapp/src/main/webapp/i18n/vivo_all_en_CA.properties:618]:
```suggestion
vis_records_start_end_of_total = Enregistrement _START_ - _END_ de _TOTAL_ 

```

File: en_CA/webapp/src/main/webapp/i18n/vivo_all_en_CA.properties (hunk 4 above)
Comment (litvinovg) [reply en_CA/webapp/src/main/webapp/i18n/vivo_all_en_CA.properties:616]: That was en_CA file

**Review by bkampe — APPROVED** [review]

**Review by chenejac — APPROVED** [review]

**Pull request merged**